hawc/apps/animal/exports.py

```diff
@@ -454,6 +457,26 @@ def _get_bmd_values(bmd, preferred_units):
             return [bmd["output"]["BMD"], bmd["output"]["BMDL"]]
         return [None, None]
 
+    @staticmethod
+    def _dose_has_n(dose_group_id: int, groups: List[Dict]) -> bool:
+        for group in groups:
+            if group["dose_group_id"] == dose_group_id:
+                return group["n"] is not None
+        return False
+
+    @staticmethod
+    def _dose_index_low_high(dose_list: List[float]) -> Tuple:
```

Comment:
I guess this is `List[Optional[float]]`, and this returns `Tuple[Optional[float], Optional[float]]` assuming these are valid.  Could you add a docstring to explain what this does?